netci.groovy

```diff
@@ -38,7 +38,7 @@ def project = 'dotnet/wcf'
             // so we set it first, and then call build.sh
             newCommitJob = job(Utilities.getFullJobName(project, jobName, false)) {
                 steps {
-                    shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:OSGroup=${os} /p:Configuration=${os}_${configuration}")
+                    shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:ShouldCreatePackage=false /p:OSGroup=${os} /p:Configuration=${os}_${configuration}")
```

Comment:
For some reason, the only command that works for me seems to be:
```
  ./build.sh /p:ShouldGenerateNuSpec=false /p:ShouldCreatePackage=false
```
I tried just ShouldCreatePackage and/or SkipBuildPackages but still got packaging errors.
If your version works for you in CI, don't block on this observation.